src/test/java/io/vertx/test/codegen/testdataobject/PropertyKindDataObject.java

```diff
@@ -0,0 +1,20 @@
+package io.vertx.test.codegen.testdataobject;
+
+import io.vertx.codegen.annotations.DataObject;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@DataObject
+public interface PropertyKindDataObject {
+  int getInteger();
```

Comment:
why are the various getters needed on this interface ? I don't see how they are used in the test

Reply:
it's a kind of PropertyKind.VALUE

Reply:
I mean the getters like getInteger(), what are their use ?

Reply:
yes, it's another way to test, but i got a new way, now i had delete it